docs/index.md

```diff
@@ -0,0 +1,39 @@
+The **Fetch Wallet**, originally forked from the [Keplr wallet](https://github.com/chainapsis/keplr-wallet), is a generic wallet for blockchains built using the Cosmos-SDK, including the Fetch network, and supports the Inter-Blockchain Communication (IBC) protocol.
+
+Some of its highlights include:
+
+- Private keys are stored locally. This removes the friction and risk of webpages having to safely manage users' private keys.
+- Because private keys are not managed by websites, users do not have to worry about their level of security.
+- The Fetch wallet can easily connect to libraries such as CosmJS, simplifying the process of connecting webpages to blockchains.
+
```

Comment:
Not happy with the highlights. @bryanchriswhite @VijaySharma815 any thoughts on the above?

Reply:
The **Fetch Wallet** originally forked from the [Keplr wallet](https://github.com/chainapsis/keplr-wallet), is a generic wallet for interacting with the Fetch blockchain network, and other cosmos-based ledgers, and performing Inter-Blockchain Communication (IBC) transactions.